cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -517,3 +517,321 @@ def test_host_comparison_multipleviews_interactions():
+        expectedResults:
+            1. Hosts view is displayed.
+            2. The summary view is displayed for the host.
+            3. The Network view is displayed.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_view_storage_adapter():
+    """
+    Testing the viewing of storage adapters.
+
+
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Click on a Host.
+            3. From the host accordion, click Properties > Storage Adapters.
+        expectedResults:
+            1. Hosts view is displayed.
+            2. The summary view is displayed for the host.
+            3. The Storage adapters view is displayed with asscoaited VMs, targets and LUNs.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_viewing():
+    """
+    Testing the viewing of host info.
+
+
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Click on a host quadicon.
+        expectedResults:
+            1. Hosts view is displayed.
+            2. The hosts summary view is displayed containing taskbar, summary panel,
+               accordion, and print/export button.
+               - Items in the accordion sections are displayed in view panel when clicked.
+               - Print/export view is displayed when print/export button is clicked.
+               - Row items that are blue on mouseover can be clicked and display additional
+                 details when done so.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_tagging():
+    """
+    Testing tagging hosts.
+
+
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Click checkbox in quadicon to select a host.
+            3. Click Edit Tags in the Policy dropdown.
+            4. Select a customer tag from the first dropdown and then a value for the tag.
+            5. Click save.
+            6. Click on the same quadicon as in step 2.
+            7. Repeat steps 1 thru 6, but select multiple hosts.
+        expectedResults:
+            1. Hosts view is displayed.
+            2.
+            3. Tag Assignement view is displayed.
+            4.
+            5. "Tag edits successfully saved." message is displayed.
+            6. Tags added in step 4 are displayed in Smart Management section of hosts summary.
+            7. Expected results are the same as steps 1 thru 6 for multiple hosts.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_drift_analysis():
```

Comment:
There are modules specific to host drift analysis and host drift, lets take a look at what validation is being done here that isn't covered in those automated tests, and add it to automation.

Reply:
I have noted that I need to look into this, but not sure of any action I can take for this PR.